WcaOnRails/spec/models/competition_spec.rb

```diff
@@ -489,18 +489,6 @@
       expect(competition.results_posted?).to be false
       expect(competition.info_for(nil)[:upload_results]).to eq "This competition is over, we are working to upload the results as soon as possible!"
     end
-
-    it "displays info if competition is in progress" do
```

Comment:
Instead of entirely deleting a test (which is never a good idea :sweat_smile: ) you should _adjust_ the test to consider the new case as well.

For your current changes, I think you can leave this test as-is (with `uses_wca_live` set to false) and create a second, almost identical test where `uses_wca_live` is set to true and you exoect the warning to display the WCA Live link instead.